.gitmodules

```diff
@@ -1,9 +1,9 @@
 [submodule "troll"]
 	path = troll
 	url = https://github.com/sonnyp/troll.git
-[submodule "src/blueprint-compiler"]
+[submodule "blueprint-compiler"]
```

Comment:
FWIW, updating _without_ changing the submodule name seems to... not work very well anyway, so might as well change it.

The right-ish way for existing users to update and migrate to the new URL seems to be, regardless of whether the submodule name changes or not:

```
git pull
git submodule deinit blueprint-compiler
git rm blueprint-compiler  # (Deletes the checked-out directory.)
git submodule init blueprint-compiler
```

The only difference is, if the name doesn't change, the `git submodule init` command **might** require `src/blueprint-compiler` as its argument. (The `deinit` works without `src/` regardless, because the directory is still there so it can match on that.)